### PR TITLE
Add office365 login

### PR DESCRIPTION
**Pull request by JoakimEdkvist**
continued on Niklas solution with login flow, managed to do a "npm install" in project folder thats why so many changes in BE prob due to updates.

not many lines of code in FE which is changed, just trying out the workflow with branches. 
=)

### CODE REVIEW

**Comment by JoakimEdkvist** [comment]
looks like the ESlint error here is due to I just commented out the "SimpleGet" component which are not in use but i wanted to keep it as a "base" to code from.

**Comment by niklas-hjelm** [comment]
Might aswell merge this and add authorization in API in a separate branch

**Review by niklas-hjelm — COMMENTED** [review]
lgtm

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -2,33 +2,49 @@ import React from 'react';
 import SimpleGet from './components/SimpleGetRequestTemplate';
```
Comment (Markussim) [inline frontend/src/App.tsx:2]: Remove this library if it's not used to fix lint

File: frontend/src/components/SimpleGetRequestTemplate.tsx [hunk 2]
```diff
@@ -1,16 +1,43 @@
 import React, { useState } from 'react';
 import axios from 'axios';
```
Comment (Markussim) [inline frontend/src/components/SimpleGetRequestTemplate.tsx:2]: Remove this library if it's not used to fix lint

**Review by Markussim — CHANGES_REQUESTED** [review]

File: frontend/src/App.tsx (hunk 1 above)
Comment (niklas-hjelm) [reply frontend/src/App.tsx]: fixed just now

File: frontend/src/components/SimpleGetRequestTemplate.tsx (hunk 2 above)
Comment (niklas-hjelm) [reply frontend/src/components/SimpleGetRequestTemplate.tsx]: fixed just now

**Review by Markussim — APPROVED** [review]
LGTM!

**Review by linusromland — APPROVED** [review]
looks good

**Pull request merged**